src/OSPSuite.Core/Domain/SensitivityAnalyses/SensitivityParameter.cs

```diff
@@ -84,41 +84,26 @@ public static IEnumerable<IBusinessRule> All
             }
          }
 
-         private static IBusinessRule parameterNotNull
-         {
-            get
-            {
-               return CreateRule.For<SensitivityParameter>()
-                  .Property(x => x.Parameter)
-                  .WithRule((sensitivityParameter, name) => sensitivityParameter.Parameter != null)
-                  .WithError((sensitivityParameter, parameter) => Error.SensitivityAnalysis.TheParameterPathCannotBeFoundInTheSimulation(sensitivityParameter.ParameterSelection.FullQuantityPath));
-            }
-         }
+         private static IBusinessRule parameterNotNull { get; } = CreateRule.For<SensitivityParameter>()
+            .Property(x => x.Parameter)
+            .WithRule((sensitivityParameter, name) => sensitivityParameter.Parameter != null)
+            .WithError((sensitivityParameter, parameter) => Error.SensitivityAnalysis.TheParameterPathCannotBeFoundInTheSimulation(sensitivityParameter.ParameterSelection.FullQuantityPath));
 
-         private static IBusinessRule nameNotEmpty
-         {
-            get { return GenericRules.NonEmptyRule<SensitivityParameter>(x => x.Name); }
-         }
+         private static IBusinessRule nameNotEmpty { get; } = GenericRules.NonEmptyRule<SensitivityParameter>(x => x.Name);
 
-         private static IBusinessRule nameUnique
-         {
-            get
+         private static IBusinessRule nameUnique { get; } = CreateRule.For<SensitivityParameter>()
```

Comment:
small optimization to create one instance instead of one every time it's asked for